formats/json/jvmTest/src/kotlinx/serialization/features/JsonJvmStreamsTest.kt

```diff
@@ -65,4 +67,22 @@ class JsonJvmStreamsTest {
             Json.decodeViaStream(String.serializer(), "\"")
         }
     }
+
+    @Test
+    fun testRandomEscapeSequences()  {
+        repeat(10_000) {
```

Comment:
The test takes 30 secs on i9, probably 10k is a bit too much for a single unit-test

Reply:
I can reduce down to 1000, should be enough